pages/s/[org]/[repo]/index.tsx

```diff
@@ -55,6 +55,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   // Cache for two hours
   context.res.setHeader("Netlify-CDN-Cache-Control", "public, max-age=0, stale-while-revalidate=7200");
+  context.res.setHeader("Netlify-Vary", "query=range");
```

Comment:
This change isn't because the page will load incorrectly for the given day range. That info is all loaded client-side. This fixes the OG image that will get cached for the page. Now it will [vary the cache](https://www.netlify.com/blog/netlify-cache-key-variations/#query) so the correct OG image link is generated for the given day range.